polaris-core/src/main/java/org/apache/polaris/core/storage/aws/AwsCredentialsStorageIntegration.java

```diff
@@ -69,13 +71,21 @@ public EnumMap<PolarisCredentialProperty, String> getSubscopedCreds(
                             allowedReadLocations,
                             allowedWriteLocations)
                         .toJson())
+                .durationSeconds(
+                    PolarisConfiguration.loadConfig(
+                        PolarisConfiguration.STORAGE_CREDENTIAL_DURATION_SECONDS))
```

Comment:
Minor suggestion: we could keep them in one line like the following if we import the static field/method.
```
.durationSeconds(loadConfig(STORAGE_CREDENTIAL_DURATION_SECONDS))
```